### PR TITLE
v4.3.0 CHANGELOG

### PR DESCRIPTION
**Pull request by oguzeroglu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,39 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `CHORE`: bump to `bpmn-js-properties-panel@0.37.1`
+
+### BPMN
+
+* `FEAT`: detection of process variables in properties panel ([#348](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/348))
+* `FEAT`: improved input/output mapping GUI component in properties panel ([#349](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/349))
+* `FEAT`: process variable overview for (sub) processes in properties panel ([#343](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/343))
+* `FEAT`: process variable typeAhead functionality in properties panel ([#345](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/345))
+* `FEAT`: adjusted element template view in properties panel ([#358](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/358))
+* `FEAT`: crop element template descriptions in properties panel ([#369](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/369))
```
Comment (pinussilvestrus) [inline CHANGELOG.md:28]: we crop all descriptions now, not only for element templates properties

**Review by MaxTru — CHANGES_REQUESTED** [review]
Please let's rewrite in the tone that we previously used:
<imperative verb> <what>

Example:
* `FEAT`: set default zoom scale when opening DMN overview ([#1848](https://github.com/camunda/camunda-modeler/issues/1848))
* `FEAT`: improve literal expression layout ([#515](https://github.com/bpmn-io/dmn-js/issues/515))

So rather than 
* `FEAT`: introduction to apply/link/unlink logic of element templates ([#1889]
(https://github.com/camunda/camunda-modeler/issues/1889))
* `FEAT`: add apply/link/unlink logic for element templates ([#1889](https://github.com/camunda/camunda-modeler/issues/1889))

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,39 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 ## Unreleased
 
 ___Note:__ Yet to be released changes appear here._
-* `FIX`: fix broken keyboard shortcuts and menu entries for Undo & Redo in decision tables ([#1843](https://github.com/camunda/camunda-modeler/issues/1843))
```
Comment (MaxTru) [inline CHANGELOG.md:8]: This was removed but not added below - did you forget that or leave out intentionally?

**Review by MaxTru — APPROVED** [review]
Just one minor finding

File: CHANGELOG.md (hunk 2 above)
Comment (oguzeroglu) [reply CHANGELOG.md:8]: Whoops, sorry my bad. Will add this now.

Comment (oguzeroglu) [reply CHANGELOG.md:8]: Done 👍

**Pull request merged**